receiver/sqlserverreceiver/scraper.go

```diff
@@ -839,9 +874,9 @@ func (s *sqlServerScraperHelper) recordDatabaseSampleQuery(ctx context.Context)
 	const dbName = "db_name"
```

Comment:
You probably want to update `cpuTime` to `cpuTimeMillisecond` to keep a consistent naming style.